ontology.ttl

```diff
@@ -839,6 +850,11 @@ Subclasses may have other ways to express the content.""" ;
                                             rdfs:subClassOf <http://odahub.io/ontology/unit#Unit> .
 
 
+### http://odahub.io/ontology/unit#PhotonUnit
+<http://odahub.io/ontology/unit#PhotonUnit> rdf:type owl:Class ;
```

Comment:
it's a photon property, not a photon unitl. Hm, maybe we can still derive it from EnergyUnit . But I fear it would not feel natural for an optical astronomer. What do you think, @juliaroquette ?

Reply:
I abandoned this PhotonUnit, introducing LengthUnit instead.

Concerning "property", this is more related to Quantity class, which we don't really use (at least my plugin doesn't) therefore it's not really well described/populated in our ontology.

Derived units are great, but it's not so trivial to express, therefore currently we have just plain list of units, grouped by "nature" but not relations between them. Extensive external ontologies exist for units, we may consider relating our units to some of them somewhere in the future.

Reply:
> I abandoned this PhotonUnit, introducing LengthUnit instead.

> Concerning "property", this is more related to Quantity class, which we don't really use (at least my plugin doesn't) therefore it's not really well described/populated in our ontology.
> 
> Derived units are great, but it's not so trivial to express, therefore currently we have just plain list of units, grouped by "nature" but not relations between them. Extensive external ontologies exist for units, we may consider relating our units to some of them somewhere in the future.

We use some IVOA ontologies where relevant. If you introduce new generic unit like `length`  it'd be good you refer to some common terms.

`nm` and `keV` are units used in expressing the same characteristic of a photon (or neutrino, etc). This should be reflected when we are adding `nm` (in addition to `nm` being simply `length`).
We need this because we are using the ontology annotations in places where this is relevant.
Maybe `Quantity` is the way to do it, or maybe there is another form association through a property I saw people use.
Do you have a proposal how to make such a link?

Reply:
I don't have a clear proposal for this. By now, most of my additions to the ontology are devoted to plugin functionality. The one you are talking about is to express more high-level concepts, plugin doesn't care if e.g. Wavelength_m is used for radio or GW.
Anyway, it's a question for another PR

Reply:
It's a question for this PR because we do not want to add dissonant change. Wavelength is related wot energy and it should be expressed.
If you have no proposal, I'll think about it next week.

Reply:
I don't think it's really dissonant, rather incomplete in some sense. 

The most comprehensive ontology of the units (and quantities) I saw is [this one](http://www.ontology-of-units-of-measure.org/resource/om-2/) as far as I remember. 
We can relate to it with owl:sameAs or other similar relation

Reply:
For me, this is dissonant, because so far we always meant keV/eV to be photon/messenger energy (energy unit, of course, is not always like that, but keV is basically always particle energy). 

Now adding distances introduces a term with ambiguous meaning, and to restore the intuition we need to add extra structures.

But ok, since I had no time since this PR was added, I agree to move it to a different [issue](https://github.com/oda-hub/ontology/issues/17).